### PR TITLE
PasswordInput: create new component

### PR DESCRIPTION
**Pull request by michac789**
https://glints.atlassian.net/browse/SS-1087?atlOrigin=eyJpIjoiMWRhNGY0YWE4M2I4NDgxOGE3MGQ4ZTlhZDA1NTExMmQiLCJwIjoiaiJ9

### CODE REVIEW

File: src/@next/PasswordInput/PasswordInputStyle.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const StyledSuffixContainer = styled.div`
+  height: 17px;
```
Comment (chiahou) [inline src/@next/PasswordInput/PasswordInputStyle.ts]: @michac789 size of a suffix icon should be 20px x 20px with 8px of paddings on the left and right
image

**Review by chiahou — COMMENTED** [review]
@michac789 according to the specs, we should allow users to see the last typed character briefly as they type on mobile devices exclusively

File: src/@next/PasswordInput/PasswordInputStyle.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import styled from 'styled-components';
 
 export const StyledSuffixContainer = styled.div`
-  height: 17px;
+  height: 20px;
+  transform: translateX(4px);
```
Comment (michac789) [inline src/@next/PasswordInput/PasswordInputStyle.ts:5]: the suffix by default has 12px padding right and 4px padding left, I can't really change that directly, so I just use transformation here... it should get me 8px padding left and 8px padding right now

**Comment by michac789** [comment]
> @michac789 according to the specs, we should allow users to see the last typed character briefly as they type on mobile devices exclusively

this one is a bit complicated, still have some problem doing it, maybe I'll try on next sprint :)

**Review by chiahou — APPROVED** [review]
looks good to me, thanks!

**Review by marthayaputra — APPROVED** [review]
LGTM

**Pull request merged**